docs/src/catalyst_functionality/network_analysis.md

```diff
@@ -204,16 +203,16 @@ complexgraph(repressilator)
 
 ![Repressilator complex](../assets/repressilator_complexgraph.svg)
 
-Here ∅ represents the empty complex, black arrows show reactions converting
+Here ∅ represent the empty complex, black arrows show reactions converting
```

Comment:
Hmm, to be honest "represents" sounds better/more correct to me.

Reply:
Ok. I will defer to you on matters of language.

Reply:
Actually, mixed up what was new and old. I agree that it should be "represents", not sure why it was changed.